### ISSUE TITLE
Cmorizing sub directories

### ISSUE THREAD

**Issue by treerink** [EC-Earth/ece2cmor3#270]
Currently sub directories like:
001
002
have to be cmorized by separate calls to ece2cmor.

Maybe nice to have the option to cmorize sub directories. Currently if the path of the directory of sub directories is specified (and no further files are in this directory) it finds no tasks, as it doesn't look in sub folders. This means that adding this feature probably does not interfere with other work flows (?).

**Comment by goord** [comment]
Almost nobody will want to do a serial ece2cmor3 run on all the legs of a climate run right? It will be terribly slow!

**Comment by tommibergman** [comment]
At least in my view, best would be to do the ece2cmor after the leg has run. In effect doing ece2cmor for previous leg in parallel with running simulation for the next leg.
